src/test/groovy/hudson/plugins/gradle/injection/MavenExamples.groovy

```diff
@@ -0,0 +1,48 @@
+package hudson.plugins.gradle.injection
+
+class MavenExamples {
```

Comment:
It looks like that this class contains different parts that constitute the maven project. Maybe to call it `MavenSnippets` or something similar?

Reply:
Done